app/gui2/src/components/GraphEditor.vue

```diff
@@ -489,6 +543,17 @@ watch(
   },
 )
 
+const componentBrowser = ref()
+const docPanel = ref()
+
+const closeCBIf = computed(() => {
```

Comment:
`computed` is not doing anything here.

Reply:
Not true, `componentBrowser` and `docPanel` refs are only populated when components are rendered, which leads to recalculation of computed.

Reply:
The computed only creates a function that reads those refs. The refs will be read when the returned function is called, but are not read by creating the function.